src/test/java/arraylist/ArrayListTest.java

```diff
@@ -0,0 +1,219 @@
+package arraylist;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class ArrayListTest {
+
+    @Test
+    public void possibleToAddStringObject() {
+        ArrayList<Object> list = new ArrayList<>();
+        list.add("test");
+        assertEquals(1, list.getCount());
+    }
+
+    @Test
+    public void possibleToAddIntegerObject() {
```

Comment:
With these tests you're basically testing Java, not your code. I wouldn't check this code with different types as you don't have anything type-related in your class. Apart from the casting in `get()` method, but that's checked by the compiler.